packages/web-components/src/components/va-memorable-date/test/va-memorable-date.e2e.ts

```diff
@@ -629,4 +629,651 @@ describe('va-memorable-date', () => {
+  });
+
+  it('swds v3 emits dateChange event when input value is updated', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(
+      '<va-memorable-date value="1999-05-03" name="test" uswds />',
+    );
+
+    const handleMonth = await page.$('pierce/[name="testMonth"]');
+    const handleDay = await page.$('pierce/[name="testDay"]');
+    const handleYear = await page.$('pierce/[name="testYear"]');
+    const spy = await page.spyOnEvent('dateChange');
+
+    await handleMonth.press('7');
+    // Click three times to select all text in input
+    await handleDay.click({ clickCount: 3 });
+    await handleDay.press('1');
+    await handleDay.press('2');
+
+    expect(spy).toHaveReceivedEventTimes(2);
+
+    // Click three times to select all text in input
+    await handleYear.click({ clickCount: 3 });
+    await handleYear.press('2');
+    await handleYear.press('0');
+    await handleYear.press('2');
+    await handleYear.press('2');
+
+    expect(spy).toHaveReceivedEventTimes(6);
+  });
+
+  it('swds v3 formats single digit days and months into 2 digits with a leading 0', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-memorable-date name="test" uswds />');
+    const date = await page.find('va-memorable-date');
+    const handleMonth = await page.$('pierce/[name="testMonth"]');
+    const handleDay = await page.$('pierce/[name="testDay"]');
+    const handleYear = await page.$('pierce/[name="testYear"]');
+    // Month
+    await handleMonth.press('1');
+    await handleMonth.press('Tab');
+    // Day
+    await handleDay.press('2');
+    await handleDay.press('Tab');
+    // Year
+    await handleYear.press('2');
+    await handleYear.press('0');
+    await handleYear.press('2');
+    await handleYear.press('2');
+    // Trigger Blur
+    await handleYear.press('Tab');
+    await page.waitForChanges();
+    expect(date.getAttribute('value')).toBe('2022-01-02');
+  });
+
+  it('swds v3 only allows specific keys to be used inside input fields', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-memorable-date name="test" uswds />');
+    const date = await page.find('va-memorable-date');
+    const handleMonth = await page.$('pierce/[name="testMonth"]');
+    const handleDay = await page.$('pierce/[name="testDay"]');
+    const handleYear = await page.$('pierce/[name="testYear"]');
+    // Month
+    await handleMonth.press('a');
+    await handleMonth.press('Tab');
+    // Day
+    await handleDay.press('b');
+    await handleDay.press('Tab');
+    // Year
+    await handleYear.press('`');
+    await handleYear.press(']');
+    await handleYear.press('/');
+    await handleYear.press(',');
+    // Trigger Blur
+    await handleYear.press('Tab');
+    await page.waitForChanges();
+    expect(date.getAttribute('value')).toBe('--');
+  });
+
+  it('swds v3 fires an analytics event when enableAnalytics is true', async () => {
```

Comment:
Most of the tests should say "uswds" and not "swds".

Are there any tests for the `monthSelect` prop?

Reply:
Great catch! I'll test the `monthSelect` prop